src/vmm/src/devices/virtio/mod.rs

```diff
@@ -22,6 +22,7 @@ mod queue;
 pub mod rng;
 pub mod test_utils;
 pub mod vhost_user;
+pub mod vhost_user_block;
```

Comment:
In commit `feat(vhost-user-block): added vhost-user-block`, if you push a newer version, please correct the typo in the commit message: s/implementatino/implementation/ .

Reply:
fixed